File: Backend/src/main/java/com/tarikturkdil/dto/ConversationResponse.java
package com.tarikturkdil.dto;

import java.time.LocalDateTime;

import lombok.Data;

@Data
public class ConversationResponse extends DtoBase {
    private UserSummaryResponse otherUser;
    private String lastMessagePreview;
    private LocalDateTime lastMessageTime;
    private long unreadCount;
}

File: Frontend/app/src/main/java/com/tarikturkdil/photoProject/data/remote/dto/ConversationResponse.kt
package com.tarikturkdil.photoProject.data.remote.dto

import kotlinx.serialization.Serializable

@Serializable
data class ConversationResponse(
    val id: Long,
    val otherUser: UserSummaryResponse,
    val lastMessagePreview: String? = null,
    val lastMessageTime: String? = null,
    val unreadCount: Long = 0,
    val createTime: String? = null
)